carbonmark-api/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "fastify-tsconfig",
   "compilerOptions": {
+    "target": "ES2020",
```

Comment:
why was this necessary?

Reply:
It was added to support this syntax for declaring bigint
const bigIntNumber = 0**n**